changelogs/fragments/1632-Validate_to_restore_keep_orginial_hlq.yml

```diff
@@ -0,0 +1,3 @@
+minor_changes:
+  - zos_backup_restore - Validate the datasets keep original hlq on restore operation.
```

Comment:
"Validate the datasets" is an extremely vague phrase. The contents of the fragment will go into the changelog and release notes and what changed will be unclear to anyone unfamiliar with this PR/code.

Maybe try something like:
> Redefines the default behavior of module option hlq. The original high level qualifiers in a backup will be used for a restore operation if no value is provided in the module option.